[2024-08-04T20:59:12.624+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-04T20:59:12.644+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.etl_task scheduled__2024-08-03T06:00:00+00:00 [queued]>
[2024-08-04T20:59:12.650+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.etl_task scheduled__2024-08-03T06:00:00+00:00 [queued]>
[2024-08-04T20:59:12.650+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-08-04T20:59:12.659+0530] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): etl_task> on 2024-08-03 06:00:00+00:00
[2024-08-04T20:59:12.665+0530] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-08-04T20:59:12.669+0530] {logging_mixin.py:188} WARNING - /Users/deepanshu/Desktop/etc-spark-airflow/env/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=67329) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-04T20:59:12.670+0530] {standard_task_runner.py:64} INFO - Started process 67372 to run task
[2024-08-04T20:59:12.670+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline', 'etl_task', 'scheduled__2024-08-03T06:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/var/folders/gw/l1rtn9c158vfbmng8xzmr2rh0000gn/T/tmpdz8uf4qg']
[2024-08-04T20:59:12.671+0530] {standard_task_runner.py:91} INFO - Job 21: Subtask etl_task
[2024-08-04T20:59:12.707+0530] {task_command.py:426} INFO - Running <TaskInstance: etl_pipeline.etl_task scheduled__2024-08-03T06:00:00+00:00 [running]> on host deepanshus-macbook-air.local
[2024-08-04T20:59:12.726+0530] {clientserver.py:505} INFO - Error while sending or receiving.
Traceback (most recent call last):
  File "/Users/deepanshu/Desktop/etc-spark-airflow/env/lib/python3.12/site-packages/py4j/clientserver.py", line 503, in send_command
    self.socket.sendall(command.encode("utf-8"))
BrokenPipeError: [Errno 32] Broken pipe
[2024-08-04T20:59:12.728+0530] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-08-04T20:59:12.728+0530] {java_gateway.py:1052} INFO - Exception while sending command.
Traceback (most recent call last):
  File "/Users/deepanshu/Desktop/etc-spark-airflow/env/lib/python3.12/site-packages/py4j/clientserver.py", line 503, in send_command
    self.socket.sendall(command.encode("utf-8"))
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/deepanshu/Desktop/etc-spark-airflow/env/lib/python3.12/site-packages/py4j/java_gateway.py", line 1038, in send_command
    response = connection.send_command(command)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/deepanshu/Desktop/etc-spark-airflow/env/lib/python3.12/site-packages/py4j/clientserver.py", line 506, in send_command
    raise Py4JNetworkError(
py4j.protocol.Py4JNetworkError: Error while sending
[2024-08-04T20:59:12.731+0530] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-08-04T20:59:12.746+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='2207deepanshu@gmail.com' AIRFLOW_CTX_DAG_OWNER='deepanshu' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='etl_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-03T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-03T06:00:00+00:00'
[2024-08-04T20:59:12.747+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-04T20:59:17.375+0530] {logging_mixin.py:188} INFO - +---+--------------------+--------------------+---------------+-----------+
| id|                name|         description|       category|avg(rating)|
+---+--------------------+--------------------+---------------+-----------+
|  1|        Sample Movie|  Sample Description|Sample Category|        4.0|
|  3|            Holidate|Holidate is a 202...|         Romcom|        3.0|
|  4|          Extraction|Extraction is a 2...|         Action|        5.0|
|  2|Avengers: Infinit...|Avengers: Infinit...|         Sci-Fi|        5.0|
+---+--------------------+--------------------+---------------+-----------+
[2024-08-04T20:59:17.377+0530] {logging_mixin.py:188} INFO - None
[2024-08-04T20:59:17.377+0530] {python.py:237} INFO - Done. Returned value was: None
[2024-08-04T20:59:17.378+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-04T20:59:17.387+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=etl_task, run_id=scheduled__2024-08-03T06:00:00+00:00, execution_date=20240803T060000, start_date=20240804T152912, end_date=20240804T152917
[2024-08-04T20:59:17.423+0530] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-04T20:59:17.439+0530] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-04T20:59:17.439+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
